src/applications/model/gaming-streaming-server.cc

```diff
@@ -56,7 +56,9 @@ GamingStreamingServer::GetTypeId (void)
 GamingStreamingServer::GamingStreamingServer ()
   : m_seq (0),
     m_totSentPackets (0),
-    m_socket (0)
+    m_totSentBytes (0),
+    m_socket (0),
+    m_sendEvent (0, EventId())
```

Comment:
you should also initialize `m_peerAddress`, `m_peerPort`, `m_packetSizeVariables`, `m_interArrivalTimesVariables`